src/plugins/simulation/SingleCellView/src/singlecellviewsimulation.h

```diff
@@ -120,6 +120,8 @@ class SingleCellViewSimulationData : public QObject
     void checkForModifications();
 
 private:
+    SolverInterface * solverInterface(const QString &pSolverName) const;
```

Comment:
Untold rule, I guess, but methods should and are to be listed after members. Here, I would put that method after the `createArrays()` and `deleteArrays()` methods.